sdk/scheduler/src/main/java/com/mesosphere/sdk/config/DefaultConfigurationUpdater.java

```diff
@@ -305,4 +310,18 @@ private void clearConfigsNotListed(Set<UUID> neededConfigs) throws ConfigStoreEx
             configStore.clear(configToClear);
         }
     }
+
+    private UpdateResult.DeploymentType getLastCompletedUpdateType() {
+        Optional<String> keyOptional = stateStore.fetchPropertyKeys().stream()
```

Comment:
Any reason to not just fetch the key you want to check? See e.g. StateStoreUtils.isSuppressed()